doc/src/manual/faq.md

```diff
@@ -627,8 +627,11 @@ have, it ends up having a substantial cost due to compilers (LLVM and GCC) not g
 around the added overflow checks. If this improves in the future, we could consider defaulting
 to checked integer arithmetic in Julia, but for now, we have to live with the possibility of overflow.
 
-In the meantime, overflow-safe integer operations can be achieved through the use of external libraries such as [SaferIntegers.jl](https://github.com/JeffreySarnoff/SaferIntegers.jl). Note that, as stated previously, the use of these libraries signically increases the execution time of code using the checked integer types. However, for limited usage, this is far less of an issue than if it were used for all integer operations. You can follow the status of the discussion [here](https://github.com/JuliaLang/julia/issues/855).
-
+In the meantime, overflow-safe integer operations can be achieved through the use of external libraries such
+as [SaferIntegers.jl](https://github.com/JeffreySarnoff/SaferIntegers.jl). Note that, as stated previously,
+the use of these libraries significally increases the execution time of code using the checked integer types.
```

Comment:
significally :)

Reply:
Yo dawg, I heard you like typos in your typo fixes...